seaborn/axisgrid.py

```diff
@@ -278,6 +278,24 @@ def legend(self):
         except AttributeError:
             return None
 
+    def tick_params(self, **kwargs):
+        """Modify the ticks, tick labels, and gridlines.
+
+        Parameters
+        ----------
+        **kwargs
```

Comment:
I think the `**` is going to mess up numpydoc formatting (I haven't build the docs to verify though).